### PR TITLE
Update assign-pod-node.md - Gt and Lt definition amendment for nodeAffinity

### PR DESCRIPTION
**Pull request by naqeebghazi**
Definitions of Gt and Lt mixed up but now amended

### CODE REVIEW

File: content/en/docs/concepts/scheduling-eviction/assign-pod-node.md [hunk 1]
```diff
@@ -634,8 +634,8 @@ The following operators can only be used with `nodeAffinity`.
 
 |    Operator    |    Behaviour    |
 | :------------: | :-------------: |
-| `Gt` | The supplied value will be parsed as an integer, and that integer is less than the integer that results from parsing the value of a label named by this selector |
-| `Lt` | The supplied value will be parsed as an integer, and that integer is greater than the integer that results from parsing the value of a label named by this selector |
+| `Gt` | The supplied value will be parsed as an integer, and that integer is greater than the integer that results from parsing the value of a label named by this selector |
+| `Lt` | The supplied value will be parsed as an integer, and that integer is less than the integer that results from parsing the value of a label named by this selector |
```
Comment (sftim) [inline content/en/docs/concepts/scheduling-eviction/assign-pod-node.md:638]:
```suggestion
| `Gt` | The field value will be parsed as an integer, and that integer is less than or equal to the integer that results from parsing the value of a label named by this selector |
| `Lt` | The field value will be parsed as an integer, and that integer is greater than or equal to the integer that results from parsing the value of a label named by this selector |
```

**Review by sftim — CHANGES_REQUESTED** [review]
/sig node

Are you 100% sure that this change is right @naqeebghazi ? I think the current page is hard to understand but also is technically accurate.

I'd welcome a change that makes the page easier to understand, but it must be correct.

File: content/en/docs/concepts/scheduling-eviction/assign-pod-node.md (hunk 1 above)
Comment (naqeebghazi) [reply content/en/docs/concepts/scheduling-eviction/assign-pod-node.md:638]: I prefer your change @sftim, let's stay with that. Thanks.

**Comment by sftim** [comment]
We don't have that kind of label (BTW this question is better taken to Slack).

**Comment by sftim** [comment]
In my opinion, this PR is ready for docs review and doesn't **need** tech review.

**Pull request merged**